packages/configs/ts/react-native.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./base.json",
+  "compilerOptions": {
+    "jsx": "react-native",
+    "skipLibCheck": true,
```

Comment:
React Native only requires `skipLibCheck` due its incompatible typing with `@types/node`, see https://github.com/DefinitelyTyped/DefinitelyTyped/issues/15960 for further context

Reply:
Too bad json doesn't support comments. But maybe we can include this info in the readme?

Reply:
convert to .ts? 😉

Reply:
What if we convert to .ts?

Reply:
@zhamujun Can you extend `.ts` files in a tsconfig?

Reply:
Found a discussion on it https://github.com/microsoft/TypeScript/issues/25271

And there's a package to do it https://www.npmjs.com/package/tsconfig.js

Maybe not worth our time to convert and introduce another dependency